### PR TITLE
Add shadow to rocket on homepage

### PR DESCRIPTION
**Pull request by pankova**
Fixes https://github.com/pushpinderpalsingh/SpaceDash/issues/25

Add shadow to rocket on the homepage:

**Before / after:**
drawing drawing

### CODE REVIEW

**Comment by pushpinderpalsingh** [comment]
Hey @pankova sorry I forget to tell you this but there is already a function I created to apply shadows to info panels. You can use see the declaration under extensions in CA Layer. I would much prefer you use that. And reduce the spread and alpha. Kinda make it like info panels! 
Also, it would be much better if you make a separate UIImage class with this functionality and make this UIImage inherit it.

**Review by pushpinderpalsingh — CHANGES_REQUESTED** [review]
Hey @pankova sorry I forget to tell you this but there is already a function I created to apply shadows to info panels. You can use see the declaration under extensions in CA Layer. I would much prefer you use that. And reduce the spread and alpha. Kinda make it like info panels!
Also, it would be much better if you make a separate UIImage class with this functionality and make this UIImage inherit it.

**Comment by pankova** [comment]
@pushpinderpalsingh hello! OK, I will fix :)

**Comment by pankova** [comment]
> Hey @pankova sorry I forget to tell you this but there is already a function I created to apply shadows to info panels. You can use see the declaration under extensions in CA Layer. I would much prefer you use that. And reduce the spread and alpha. Kinda make it like info panels!
> Also, it would be much better if you make a separate UIImage class with this functionality and make this UIImage inherit it.

Hello! But I use the applySketchShadow method from CALayer extension, as InfoPanelView in init(). I checked the actual code in develop and master - there are no more wrappers of this functionality

**Comment by pushpinderpalsingh** [comment]
I think you can just use {uiImageviewName}.layer.applySketchShadow() anywhere in thw project

**Comment by pankova** [comment]
@pushpinderpalsingh in my opinion, default params are not suitable for the rocket image. 

Even _InfoPanelView_ calls _applySketchShadow_ with custom parameters. And these parameters are not OK for the rocket (again - only in my opinion).
So I couldn't call applySketchShadow() without arguments.

But if you wish, I will use the default 🙂

**Comment by pushpinderpalsingh** [comment]
I never said to use the default parameters use whatever you want but don't repeat the functions already present in the codebase.

**Comment by pankova** [comment]
@pushpinderpalsingh hello! I have made changes and waiting for your opinion 🙂

File: SpaceDash/Controller/UIView/RocketImageView.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//
+//  RocketImageView.swift
```
Comment (pushpinderpalsingh) [inline SpaceDash/Controller/UIView/RocketImageView.swift:2]: Place this file under ImageView.

**Review by pushpinderpalsingh — CHANGES_REQUESTED** [review]
Your pr is perfect, just place the RocketImageView file under ImageView.

File: SpaceDash/Controller/UIView/RocketImageView.swift (hunk 1 above)
Comment (pankova) [reply SpaceDash/Controller/UIView/RocketImageView.swift]: Done👌 And I squashed all commits in one. Thanks! 🙂

**Comment by pushpinderpalsingh** [comment]
Thanks for your contribution. @all-contributors add @pankova for code

**Comment by pankova** [comment]
@pushpinderpalsingh hello! Could you please add 'hacktoberfest' label, I could see this PR in my progress? 🙂

**Comment by pankova** [comment]
Hello! Could you please label PR as 'hacktoberfest-accepted', I could see it in my progress 🙂
```
Individual pull requests can also be directly opted-in to Hacktoberfest with the ‘hacktoberfest-accepted’ label, on any public GitHub repository, without the need for the ‘hacktoberfest’ repository topic.
```
[(c) updated rules](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

**Comment by pushpinderpalsingh** [comment]
Sure thing, no issues!

**Pull request merged**